### ISSUE TITLE
Nested Collections Parent/Child

### ISSUE THREAD

**Issue by markprouse** [mattbrailsford/umbraco-fluidity#118]
Is it possible to have nested collections with Fluidity?

Persons
____|--> Person (fields etc)
____|--> Children
____________|--> Child (fields etc)
____________|--> Children
____________________|--> Child (fields etc)

If not natively, is there a work around which i can do achieve this using say three Collections so as i drill down each time i see the Children list and tabs for fields etc?

**Comment by mattbrailsford** [comment]
@markprouse not currently no, it's a long standing open issue #41. I'm hoping to spend some time on Fluidity next year for some big updates, but until I can focus on it, that won't be possible.

The only thing you can do right now is have them as separate collections and give the child items a parent picker property to pick the master entity. Then in the list view, created filters to filter by specific parent types. Not ideal, but best we have atm.

**Comment by c9mb** [comment]
If you haven't already done so, then I'd recommend that before starting on big updates to Fluidity, you should download a trial copy of Sitefinity and check out the 'Dynamic Modules' function, for some inspiration in what Fluidity could be to Umbraco. Prior to being an Umbraco developer, I spent many years as a Sitefinity developer, and it was quite a step backwards to lose the functionality of something I'd simply taken for granted as a basic CMS feature. Of course, commercial software vs. open source is a challenge when it comes to matching feature-sets, and everyone makes their choices on where that trade-off needs to be, but it's good to know what's available in the marketplace and where the bar is set.
